modules/reference/pages/public-metrics-reference.adoc

```diff
@@ -668,6 +668,84 @@ Total number of requests that uploaded an object to cloud storage.
 
 ---
 
+[[tls_metrics]]
+== TLS metrics
+
+=== redpanda_tls_truststore_expires_at_timestamp_seconds
+
+The expiration time, represented as a unix-style timestamp, of the shortest-lived CA in the installed CA chain. This lists all resources that have at least one CA configured.
+
+*Type*: gauge
+
+*Labels*:
+
+* `area`
+* `detail`
+
+*Usage*: This metric is recorded for all resources that may have a TLS certificate installed. The `area` and `detail` labels provide a hierarchical way of identifying a single resource you need information on.
+
+---
+
+=== redpanda_tls_certificate_expires_at_timestamp_seconds
+
+The expiration time, represented as a unix-style timestamp, for the shortest-lived installed certificate. While up to five certificates may be installed for a resource, this provides only the timestamp of the next to expire. This lists all resources that have at least one certificate installed.
+
+*Type*: gauge
+
+*Labels*:
+
+* `area`
+* `detail`
+
+*Usage*: This metric is recorded for all resources that may have a TLS certificate installed. The `area` and `detail` labels provide a hierarchical way of identifying a single resource you need information on.
+
+---
+
+=== redpanda_tls_certificate_serial
+
+The serial number of the shortest-lived installed certificate. As these serial numbers are technically unbounded, this will return the least significant 4B characters of the certificate's serial number. While up to five certificates may be installed for a resource, this provides only the serial number of the next to expire. This lists all resources that have at least one certificate installed.
```

Comment:
```suggestion
The serial number of the shortest-lived installed certificate. As these serial numbers are technically unbounded, this will return the least significant 4B characters of the certificate's serial number. While up to five certificates may be installed for a resource, this provides only the serial number of the next certificate to expire. This lists all resources that have at least one certificate installed.
```